src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -151,9 +160,23 @@ public ObservableList<Delivery> getFilteredDeliveryList() {
     @Override
     public void updateFilteredDeliveryList(Predicate<Delivery> predicate) {
         requireNonNull(predicate);
+        isViewingFilteredList = true;
```

Comment:
The use of a boolean field to determine which list to display is brillaintly designed!